manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Schinagl.LinkShellextension
+PackageVersion: 3.9.3.5
+PackageLocale: en-US
+Publisher: Schinagl
+PackageName: Link Shellextension

```

```suggestion
PackageName: Link Shell Extension
```
This must match Control Panel / Apps&Features